File: pjecalc-fonte/negocio/br/jus/trt8/pjecalc/negocio/comum/validators/calculo/Erro.java
/*
 * Decompiled with CFR 0.152.
 */
package br.jus.trt8.pjecalc.negocio.comum.validators.calculo;

import br.jus.trt8.pjecalc.negocio.comum.validators.calculo.MsgValidador;

public class Erro
extends MsgValidador {
    private static final long serialVersionUID = -8274501333296240441L;
    private static final String TIPO = "ERRO";

    @Override
    public boolean isImpeditivo() {
        return true;
    }

    @Override
    public String getTipo() {
        return TIPO;
    }
}

File: src/lib/pjecalc/core/comum/validators/calculo/erro.ts
/**
 * PJe-Calc v2.15.1 — Erro
 * Porte de: br.jus.trt8.pjecalc.negocio.comum.validators.calculo.Erro
 */
import { MsgValidador } from './msg-validador';

export class Erro extends MsgValidador {
  private static readonly TIPO = 'ERRO';
  isImpeditivo(): boolean { return true; }
  getTipo(): string { return Erro.TIPO; }
}
